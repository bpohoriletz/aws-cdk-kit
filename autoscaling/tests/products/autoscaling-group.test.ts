import AutoscalingGroupProduct from '../../products/autoscaling-group';

describe('new AutoscalingGroupProduct()', () => {
  let instance: AutoscalingGroupProduct;
  beforeEach(() => {
    instance = new AutoscalingGroupProduct();
  });

  it('matches snapshot', () => {
    expect(instance).toMatchSnapshot();
  });
});
