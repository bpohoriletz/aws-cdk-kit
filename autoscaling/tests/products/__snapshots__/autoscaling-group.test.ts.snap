// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new AutoscalingGroupProduct() matches snapshot 1`] = `AutoscalingGroupProduct {}`;
